### ISSUE TITLE
Translations

### ISSUE THREAD

**Issue by CXY23** [slapperwan/gh4a#986]
Can developers add a language interface change function?This software is very easy to use

**Comment by ccuser44** [comment]
What? What does a proxy have to do with language support?

**Comment by sebastien46** [comment]
duplicate of #902 & #947 for 1st part
as for 2nd part, github is passively blocked by Chinese Firewall, so a VPN connection is heavily recommended
(& afaik entirely supported by this app?)

**Comment by maniac103** [comment]
Dupe of #947
